recipes/xdrfile/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "xdrfile" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.gromacs.org/pub/contrib/{{ name }}-{{ version }}.tar.gz
+  sha256: e3c587c5ff24441a092fe2f3bc1dc03667bf126558f437161e779bfbcce48022
+
+build:
+  number: 0
+  skip: True  # [win]
```

If a package uses this will it always be needed at runtime (i.e. it only includes a shared library)? If so you should use [`run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements):
```suggestion
  skip: True  # [win]
  run_exports:
    - {{ pin_subpackage('xdrfile', max_pin='x.x.x') }}
```
You should adjust `x.x.x` to be your expected ABI stability. (If I compile something downstream against `1.1.4` can I expect the same binaries to work with `1.1.5`? `1.2`? `2.0`?)

---

@chrisburr thanks for your feedback! About the compatible version, I would conservatively pin to the precise version (`pin_subpackage('xdrfile', min_pin='x.x.x', max_pin='x.x.x')`). I don't think one can find other versions around anyway, and it is not updated since a while. What do you think? Thanks again!